lib/property_table.ex

```diff
@@ -49,6 +49,19 @@ defmodule PropertyTable do
   * `:tuple_events` - set to `true` for change events to be in the old tuple
     format. This is not recommended for new code and hopefully will be removed
     in the future.
+
+  Persistent options:
+
+  * You MUST set at least `:persist_data_path` for any of the other options to be respected!
+  * If the table can restore its data from disk, it will IGNORE your initial `:properties` value.
+
+  * `:persist_data_path` - set to a directory where PropertyTable will
+    persist the contents of the table to disk, snapshots will also be stored here.
+  * `:persist_interval` - if set PropertyTable will persist the contents of
+    tables to disk in intervals of the provided value (in milliseconds) automatically.
+  * `:persist_max_snapshots` - Maximum number of manual snapshots to keep on disk before they
+    are replaced - (oldest snapshots are replaced first.)
+  * `:persist_compression` - `0..9` range to compress the terms when written to disk, see `:erlang.term_to_binary/2`
```

Comment:
```suggestion
  * `:persist_max_snapshots` - Maximum number of manual snapshots to keep on disk before they
    are replaced - (oldest snapshots are replaced first.) Defaults to 25.
  * `:persist_compression` - `0..9` range to compress the terms when written to disk, see `:erlang.term_to_binary/2`. Defaults to 6.
```

Did I get the defaults right? Also, `:persist_interval` defaults to `:infinity` or off, right?

Reply:
That's correct, it will not periodically persist if you don't define an interval, you will have to call the flush to disk method by hand whenever you want to write the table